src/org/daisy/dotify/formatter/impl/volume/TocSequenceEventImpl.java

```diff
@@ -146,10 +149,10 @@ public BlockSequence getBlockSequence(FormatterContext context, DefaultContext v
 
                 case DOCUMENT: {
                     for (int vol = 1; vol <= crh.getVolumeCount(); vol++) {
-                        final int volumeFirstContentPage = crh.getPageNumberOfFirstContentPageOfVolume(vol);
+                        final int v = vol;
                         Collection<Block> volumeToc = data.filter(
                                 refToVolume(vol, crh), rangeToVolume(vol, crh),
-                                volumeFirstContentPage
+                                () -> crh.getPageNumberOfFirstContentPageOfVolume(v)
```

Comment:
... and here change `v` to `vol`

Reply:
I need a variable that is declared final.

Reply:
Yeah... that's correct